core/src/cli_commands/vote_update_pool.rs

```diff
@@ -323,7 +325,7 @@ mod tests {
             new_pool_box_address_hash,
             new_reward_token_id,
             100_000,
-            height - 3,
+            BlockHeight(height.0 - 3), // TODO: should BlockHeight implement Sub for arbitrary units?
```

Comment:
What do you think?

Reply:
Sounds nice! If it does not take too much effort. :) I mean, bounds checks are helpful, but the main reason for this newtype is API.